index.html

```diff
@@ -1567,7 +1567,7 @@ <h5>
             <p its-locale-filter-list="en" lang="en">Interpuncts apply to Chinese only. When a translated foreign name contains a Latin character, a western period should be used rather than a interpunct. For example, <span lang="zh-hant">比尔·盖茨</span> but <span lang="zh-hant">B. 盖茨</span>.</p>
             <p its-locale-filter-list="zh-hans" lang="zh-hans">间隔号为中文标点，当外语姓名包含西文首字母缩写作为单节时，原则上西文字母后面应该使用西文句点，如“比尔·盖茨”、“B. 盖茨”。</p>
             <p its-locale-filter-list="zh-hant" lang="zh-hant">間隔號為中文標點，當外語姓名包含西文首字母縮寫作為單節時，原則上西文字母後面應該使用西文句點，如「比爾·蓋茨」、「B. 蓋茨」。</p>
-            <p its-locale-filter-list="en" lang="en">The width of interpuncts varies in different regions. In principle, in Hong Kong and Taiwan, interpuncts should have the same dimensions as a Hanzi in both vertical writing mode and horizontal writing mode. In Mainland China, interpuncts take up half the space of a Hanzi. </p>
+            <p its-locale-filter-list="en" lang="en">The width of interpuncts varies in different regions. In principle, in Hong Kong and Taiwan, interpuncts should have the same dimensions as a Hanzi in both vertical writing mode and horizontal writing mode. In Mainland China, interpuncts take up half the space of a Hanzi.</p>
```

Comment:
The English translation of 'hanzi' (which is in the Chinese language) is 'Han character'.

Reply:
Agree. #496 & #504 are the related issues/PRs.

Reply:
There are still places where 'hanzi' is used after the PR being merged, so we probably need to check again.